packages/taquito/src/operations/operation-emitter.ts

```diff
@@ -178,6 +178,13 @@ export abstract class OperationEmitter {
               ...getSource(op),
               ...getFee(op),
             };
+          case OpKind.SMART_ROLLUP_ORIGINATE:
+            return {
+              ...op,
+              ...getSource(op),
+              ...getFee(op),
+              storage_limit: op.storage_limit.toString(),
```

Comment:
Thought getFee function is taking care of storage_limit too

Reply:
```
const getFee = (op: RPCOpWithFee) => {
      const opCounter = ++counters[publicKeyHash];
      return {
        counter: `${opCounter}`,
        fee: typeof op.fee === 'undefined' ? '0' : `${op.fee}`,
        gas_limit: typeof op.gas_limit === 'undefined' ? '0' : `${op.gas_limit}`,
        storage_limit: typeof op.storage_limit === 'undefined' ? '0' : `${op.storage_limit}`,
      };
    };
```
have to dig deeper but this made the difference because for som reason its still being seen as undefined